doc/ApiDocGeneration/Generate-Api-Docs.ps1

```diff
@@ -60,9 +74,10 @@ $YamlOutDir = "${BinDirectory}/${ArtifactsDirectoryName}/dll-yaml-output"
 $DocOutDir = "${BinDirectory}/${ArtifactsDirectoryName}/docfx-output/docfx_project"
 $DocOutApiDir = "${DocOutDir}/api"
 $DocOutHtmlDir = "${DocOutDir}/_site"
+echo "##vso[task.setvariable variable=DocHtmlDir]$DocOutHtmlDir"
```

Comment:
We won't need this output variable if we move all the mashup steps into this script.

Reply:
Done.